### ISSUE TITLE
Cannot build iOS

### ISSUE THREAD

**Issue by irekrog** [numandev1/react-native-keys#40]
### Current behavior

```
Run pre-actions
Build | Scheme MyProject
Run custom shell script 
SchemeAction Run\ Script /var/folders/qp/ygqy3bwj78v_8vp2nhfs_2hh0000gn/T/SchemeScriptAction-SDT73q.sh
    cd /var/folders/qp/ygqy3bwj78v_8vp2nhfs_2hh0000gn/T/
...
env: node: No such file or directory
```

### Expected behavior

Should work fine

### Platform

- [X] iOS

### React Native Version
0.71.7

**Comment by numandev1** [comment]
can you share the pre-action script of the current scheme?🙏
![image](https://github.com/numandev1/react-native-keys/assets/36044436/ff5945e0-e123-463d-aaa6-601c0ecbb5b6)

**Comment by irekrog** [comment]
Sure
![screenshot](https://github.com/numandev1/react-native-keys/assets/11172548/e63651b7-c9b6-4918-bd8d-5729c1a40593)

**Comment by numandev1** [comment]
@irekrog can you check `keysIos.js` file exists in node_modules or not?

**Comment by numandev1** [comment]
@irekrog  🙏one more thing, can you add this line before `export KEYFILE=env.staging.json`
```
exec > ${PROJECT_DIR}/prebuild.log 2>&1
```
it will create `prebuild.log` at the root of your project where we can see the log

**Comment by irekrog** [comment]
> @irekrog 🙏one more thing, can you add this line before `export KEYFILE=env.staging.json`
> 
> ```
> exec > ${PROJECT_DIR}/prebuild.log 2>&1
> ```
> 
> it will create `prebuild.log` at the root of your project where we can see the log

ok, I have added and in logs is:
```
env: node: No such file or directory
```

**Comment by numandev1** [comment]
@irekrog did you make `.env.staging.json` at the root of your project?

**Comment by irekrog** [comment]
@numandev1 
yes, I have this file `env.staging.json` at the root, should starts with dot `.env.staging.json`?

**Comment by numandev1** [comment]
@irekrog I think dot does not matter, may we both debug it through Zoom?

**Comment by irekrog** [comment]
@numandev1 

One more question

Your library should work fine with M1?

**Comment by numandev1** [comment]
Sound good ❤ I am little busy, I will add nvm into the docs

**Comment by numandev1** [comment]
@irekrog yes, it works fine with M1

**Comment by numandev1** [comment]
I am reopening this issue as it is not closed for all, it will remember me to add nvm into docs

**Comment by numandev1** [comment]
closed by #42
